packages/dso-toolkit/src/components/skiplink/skiplink.models.ts

```diff
@@ -0,0 +1 @@
+export interface Skiplink {}
```

Comment:
Dit object is het template model en dient als technische lijm tussen het visuele en functionele ontwerp en de technische implementaties.

Kijkende naar het Skiplink component denk ik dat er twee properties zijn:

```suggestion
export interface Skiplink {
  to: string;
  label: string;
}
```

Reply:
@hansgrimm Hier zal ik mij over ontfermen.

Reply:
Skiplink model geïmplementeerd.